optimade/server/run.py

```diff
@@ -0,0 +1,6 @@
+import uvicorn
+from optimade.server.main import app
+
+if __name__ == '__main__':
```

Comment:
Why have this file, when we already have the bash file? Is it because you are working in a Windows environment?

Reply:
I can remove it without any problem. I just use Pycharm so the debugging and profiling is much easier if you can run the server directly instead of using a cli.